plugins/polio/models.py

```diff
@@ -1054,6 +1054,65 @@ class VaccineArrivalReport(SoftDeletableModel):
     objects = DefaultSoftDeletableManager()
 
 
+class VaccineStock(models.Model):
+    account = models.ForeignKey("iaso.account", on_delete=models.CASCADE, related_name="vaccine_stocks")
+    country = models.ForeignKey(
+        "iaso.orgunit",
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="vaccine_stocks",
+        help_text="Unique (Country, Vaccine) pair",
+    )
+    vaccine = models.CharField(max_length=5, choices=VACCINES)
```

Comment:
Shouldn't there also be a link with the VAR from "SupplyChain", or is it enough with the country to retrieve them?

I also wonder about the dashboards and the long term: we have a chance to have the same sort of problems as preparedness, i.e: the data will just grow and grow for each country+vaccine and will take longer an longer to compute in the API. We should probably have some way of archiving results after a year or something and use the last year's balance as a starting point for the computation (like any accounting software I guess). This is probably another ticket to put in specs, but I think it's good to already have that in mind